content/_help/specific-agencies/ssa._en.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: help
+title: SSA.gov
+category: specific-agencies
+permalink: /help/specific-agencies/ssa/
+keywords: social security, social security benefits, replacement social security card, social security card, ssa, social security card replacement, social security statement, replace social security card, new social security card, new card, replacement card, ss statement, social security account, social security administration, ssa.gov, i need a new social security card
+order: 3
+redirect_from:
```

Comment:
We can remove this across all three files if we're not redirecting from anywhere.

Although in the case of the English page, I think we'll want to set up a redirect from the `/en/` prefix, like we do with  other pages.

```suggestion
redirect_from:
  - /en/help/specific-agencies/ssa/
```